.vib/rabbitmq/cypress/cypress/integration/rabbitmq_spec.js

```diff
@@ -0,0 +1,109 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#versions')
+    .invoke('text')
+    .should('match', /RabbitMQ\s\d+\.\d+\.\d+/);
```

Comment:
I'd say to use a more resilient RegEx. 

```suggestion
    .should('match', /RabbitMQ.*\d+\.\d+\.\d+/);
```

If in the next version, the version is printed as `RabbitMQ: 9.8.2`, this will still match.

Reply:
Improved as per suggestion. Thanks!